addons/TBMod_persistence/functions/fn_getSavedNamesFromServer.sqf

```diff
@@ -0,0 +1,28 @@
+#include "../script_macros.hpp"
+/*
+    Part of the TBMod ( https://github.com/TacticalBaconDevs/TBMod )
+    Developed by http://tacticalbacon.de
+*/
+GVAR(transfer) = false;
+publicVariable QGVAR(transfer);
+
+[[ACE_player], 
+{
+    params ["_target"];
+
+    _transferData = profileNamespace getVariable [QGVAR(savedNames), []];
+
+    [[_transferData], 
+    {
+        params ["_data"];
+
+        GVAR(savedNamesServer) = _data;
+
+        GVAR(transfer) = true;
+        publicVariable QGVAR(transfer);
+    }] remoteExec ["call", _target];
```

Comment:
wenn schon das ganze in unscheduled env execed werden soll, dann kann man auch **remoteExecCall** nehmen

Reply:
@Er1807 dem zweiten call kannste per cmd geben, von dem es kamm per `remoteExecutedOwner`